src/lib/actions/RunAction.ts

```diff
@@ -0,0 +1,8 @@
+
+
+export class RunAction {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	public execute(input: object): void {
```

Comment:
I recommend instead of suppressing these lines with comments... to just update the rule with an argsIngorePattern as I do here: https://github.com/forcedotcom/code-analyzer-core/blob/dev/packages/code-analyzer-core/eslint.config.mjs

```
    {
        rules: {
            "@typescript-eslint/no-unused-vars": ["error", { "argsIgnorePattern": "^_" }]
        }
    }
```

Then you can just have public execute(_input: object) for now. That is, the underscore will be sufficient since this is a common thing.